core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -189,6 +192,10 @@ object BootMonitors extends LazyLogging {
     system.actorOf(BucketDeletionMonitor.props(slickDataSource, gcsDAO, 10 seconds, 6 hours))
   }
 
+  private def startWorkspaceBillingAccountMonitor(system: ActorSystem, slickDataSource: SlickDataSource, gcsDAO: GoogleServicesDAO)(implicit cs: ContextShift[IO]) = {
+    system.actorOf(WorkspaceBillingAccountMonitor.props(slickDataSource, gcsDAO, 60 seconds, 6 hours))
```

Comment:
probably should parameterize the `6 hours` number and ultimately load it from config in Boot

Reply:
While I am here, should I also parameterize the BucketDeletionMonitor above?

Reply:
nah... it is going away and I doubt we will want to tune it.